thunder/executors/nvfuserex_impl.py

```diff
@@ -479,6 +484,9 @@ def create_fusion_definition_wrapper(
     #   objects is captured by names in the trace.
     # These properties are distinct from the inputs and outputs to the trace itself, which
     #   may contain duplicates and whose order must be preserved.
+    store_inputs: None | bool = get_compile_option(
+        "nv_store_fusion_inputs", "Enable nvFuser store fusion inputs for repro."
```

Comment:
"Enable nvFuser to store fusion inputs for repro" or "Let nvFuser store fusion inputs for repro"?

Reply:
Yeah I red the descriptions of the other nv compile options and I went for something similar but it better to phrase it differently